apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -107,7 +107,7 @@ const CreateEventPage = ({ history }: CreateEventProps) => {
         } = await uploadFile({ variables: { file: file } });
         await createEvent({
           variables: {
-            createEventInput: {
+            input: {
```

Comment:
Not a big fan of this vague `input`, could we do `event` or `eventInput`?

Reply:
I find it redundant since you should already know what the query is. It's the input to the create event query, idk if we need to mention that it's the create event input or event input.

Reply:
If you insist that being explicit about it helps then I don't mind

Reply:
I don't insist, if you'd prefer to keep it input it's okay too. I just felt it was easier to read

Reply:
Maybe instead of `input`, `event` could be used? I'm not a fan of putting `createEventInput` or something similarly long, but `event` clearly shows that what you need to give as input is an event, or at least part of an event

Reply:
Sounds good to me